rs/nns/test_utils/src/gtc_helpers.rs

```diff
@@ -242,7 +242,7 @@ fn make_neuron(
 
     let (dissolve_state, aging_since_timestamp_seconds) = if dissolve_delay_seconds == 0 {
         (
-            Some(DissolveState::WhenDissolvedTimestampSeconds(0)),
+            Some(DissolveState::WhenDissolvedTimestampSeconds(1)),
```

Comment:
This means it's dissolved 1 second after linux epoch.  Should make no functional difference.